# What does the following error msg tell you?...

# ArgumentError: wrong number of arguments (1 for 2)
# from (irb):1:in `calculate_product'
# from (irb):4
# from /Users/username/.rvm/rubies/ruby-2.0.0-p353/bin/irb:12:in `<main>'

puts "It's telling me that I called a method using an incorrect number of arguments."
puts "This happened in the method 'calculate_product' on line 1 of the code."
puts "The incorrect arg is used on line 4"